fs/copy.ts

```diff
@@ -0,0 +1,206 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as path from "./path/mod.ts";
+import { ensureDir, ensureDirSync } from "./ensure_dir.ts";
+
+export interface CopyOptions {
+  /* overwrite existing file or directory, default is false */
+  overwrite?: boolean;
+}
+
+function isSrcSubdir(src: string, dest: string): boolean {
+  const srcArray = src.split(path.sep);
+  const destArray = dest.split(path.sep);
+
+  return srcArray.reduce((acc, current, i) => {
+    return acc && destArray[i] === current;
+  }, true);
+}
+
+/* copy file to dest */
+async function copyFile(src: string, dest: string): Promise<void> {
+  await Deno.copyFile(src, dest);
+}
+
+function copyFileSync(src: string, dest: string): void {
+  Deno.copyFileSync(src, dest);
+}
+
+/* copy link to dest */
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+async function copyLink(src: string, dest: string): Promise<void> {
+  throw new Error("Not implemented copy link");
```

Comment:
Link is not supported in the current

Reply:
use unimplemented() from testing/asserts.ts